shard.yml

```diff
@@ -1,7 +1,7 @@
 name: lucky
 version: 0.27.2
 
-crystal: ">=0.36.1, < 2.0.0"
+crystal: ">=1.0.0"
```

Comment:
I removed the upper limit because of this forum post https://forum.crystal-lang.org/t/artificial-shard-restrictions-version-2-0-0-considered-harmful/3319